src/DSL/MarkdownBuilder.cs

```diff
@@ -234,7 +234,7 @@ public void Console(params string[] commands)
         /// <param name="commands">The commands that will be output as individual lines within the console section.</param>
         public void Console(IEnumerable<string> commands)
         {
-            Block(commands);
+            Block("bash", commands);
```

Comment:
Why this change?

Reply:
Before this change our generated console blocks in markdown files didn't have a language specified. I don't know if this will actually make a change in our console blocks, but I figured its better to have it there just in case the text coloring/highlighting improves in the future.